vars/dockerBuildArgs.groovy

```diff
@@ -49,26 +101,43 @@ String call(Map config = [:]) {
       ret_str += " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR)
     }
 
-    if (add_repos) {
-      if (env.REPOSITORY_URL) {
-        ret_str += ' --build-arg REPO_URL=' + env.REPOSITORY_URL
-      }
-      if (env.DAOS_STACK_EL_7_LOCAL_REPO) {
-        ret_str += ' --build-arg REPO_EL7=' + env.DAOS_STACK_EL_7_LOCAL_REPO
+    String repo_base = env.REPOSITORY_URL
```

Comment:
Is there a reason to access environment variables indirectly through local variable assignments?  I don't think there is any extra cost associated with accessing them directly and accessing them directly instead of through a local variable is more clear I think rather than having to remember (i.e. while reading) yet another variable's value.